ibc-clients/src/lib.rs

```diff
@@ -16,3 +16,9 @@ pub mod tendermint {
     #[doc(inline)]
     pub use ibc_client_tendermint::*;
 }
+
+/// Re-exports implementations of ICS-08 Wasm light client types.
+pub mod wasm_types {
+    #[doc(inline)]
+    pub use ibc_client_wasm_types::*;
```

Comment:
Right! At the moment, we're not going to add a Wasm light client implementation on top of the types. Since we're diving into ibc-go driven chains, we just need those types to get compatible with their Wasm proxy client.